examples/rust/objectron/build.rs

```diff
@@ -54,6 +54,7 @@ fn main() -> Result<(), std::io::Error> {
 }
 
 /// Only touch the file if the contents has actually changed
+// TODO(cmc): use the same source tracking system as re_types*
 fn write_file_if_necessary(
```

Comment:
Move it into `re_build_build_info` as well?

Reply:
I was hesitant to have the objectron example depends on re_build_info... but then again I guess it's just the build-dependencies, let's go for it